python/revng/scripting/projects/daemon_project.py

```diff
@@ -0,0 +1,182 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import json
+from typing import Dict, List
+
+import yaml
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+
+from .project import Project
+
+from revng.model import Binary, Function, dump_diff_to_string, load_model_from_string, make_diff  # type: ignore[attr-defined] # noqa: E501 # isort: skip
```

Comment:
Why is the `isort: skip` needed? I'd expect it to duplicate the comments but not much

Reply:
Because using `force-formatting` breaks up the lines but it doesn't preserver the comments so for example this is a diff created by `isort`
```
  - from revng.model import load_model_from_string, make_diff  # type: ignore[attr-defined]
  + from revng.model import load_model_from_string, make_diff
```
This the causes `mypy` to fail.